seatunnel-connectors/seatunnel-connector-spark-hudi/src/main/scala/org/apache/seatunnel/spark/sink/Hudi.scala

```diff
@@ -41,7 +41,7 @@ class Hudi extends SparkBatchSink {
   override def output(df: Dataset[Row], environment: SparkEnvironment): Unit = {
     val writer = df.write.format("org.apache.hudi")
     for (e <- config.entrySet()) {
-      writer.option(e.getKey, e.getValue.toString)
+      writer.option(e.getKey, config.getString(e.getKey))
```

Comment:
Change to `String.valueOf(e.getValue.unwrapped())` is better.

Reply:
ok